public/views/partials/molecules/chip.liquid

```diff
@@ -42,7 +44,11 @@ metadata:
   assign icon_params = '{}' | parse_json | hash_merge: name: 'x', class: 'w-3 h-3 ml-2 '
 %}
 <div class="inline-block {{wrapper_classes}}">
-  <{{tag}} class="{{classes}}">
+  <{{tag}} class="{{classes}}"
+    {% for attribute in attributes %}
+      {{ attribute[0] | html_safe }}="{{ attribute[1] | html_safe }}"
```

Comment:
do you need html_safe? any explicit html_safe is a red flag and potential XSS injection ; most likely won't be an issue here, as attributes come from developer, just saying :)

Reply:
btw - you might want to also use `{% print attribute[0] %}` instead of `{{ attribute[0] }}` - should be faster, because `print` just outputs whatever there is without sanitization/escaping